### PR TITLE
Add order to pagination params

### PR DESCRIPTION
**Pull request by pliao28**

### CODE REVIEW

File: workos/directory_sync.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 from workos.utils.validation import DIRECTORY_SYNC_MODULE, validate_settings
 
 RESPONSE_LIMIT = 10
+ORDER_DEFAULT = "normal"
```
Comment (hadihallak) [inline workos/directory_sync.py:10]: Just to be consistent across SDKs I think we should make the default whatever the equivalent for undefined in python

**Review by hadihallak — COMMENTED** [review]
I touched python once or twice in my whole life so I'm not sure if we are able to type values this way but it would be nice if we could type the available values somewhere `asc` | `desc`

**Comment by hadihallak** [comment]
not sure how optional args work in python but I think we might need to move the order param to be the last param like @sheldonvaughn suggested in the SDK PR for PHP to make sure that this isn't a breaking change.

**Comment by pliao28** [comment]
> not sure how optional args work in python but I think we might need to move the order param to be the last param like @sheldonvaughn suggested in the SDK PR for PHP to make sure that this isn't a breaking change.

I don't think param ordering conventions matter much for Python, as they use assignment operators, but I will change the ordering across the board just to match for consistency. Originally I was thinking of setting it after limit because that would make the most sense when thinking about pagination, where you would set limit and order first

**Comment by hadihallak** [comment]
Yeah I totally agree. I also think that the new planed methods for pagination would probably render these params obsolete so I totally agree with you.

**Pull request merged**